wp-content/themes/core/components/blocks/gallery_grid/js/gallery_grid.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * @module
+ * @description JavaScript that drives Dialog
+ */
+
+import A11yDialog from 'mt-a11y-dialog';
+import * as tools from 'utils/tools';
+import { trigger } from 'utils/events';
+
+const el = {
+	containers: tools.getNodes( '[data-js="dialog-trigger"]', true, document, true ),
```

Comment:
This trigger is pretty generic. Either it should apply to all dialogs (and be init'd in the dialog component), or the trigger should be specific to the gallery grid block.